docs/release_notes/sat_2.6_release_notes.md

```diff
@@ -0,0 +1,73 @@
+# Changes in SAT 2.6
+
+The 2.6.4 version of the SAT product includes:
+
+- Version 3.24.0 of the `sat` python package and CLI.
+- Version 2.0.0-1 of the `sat-podman` wrapper script.
+- Version 1.6.1 of the `sat-install-utility` container image.
+
+## New `sat` commands
+
+No new `sat` commands were added in SAT 2.6.
+
+## Changes to `sat bootsys`
+
+- Functionality has been added to the `platform-services` and `cabinet-power`
+  stages of `sat bootsys boot` which allows SAT to automatically re-create
+  Kubernetes CronJobs which may have become stuck during shutdown, boot, or
+  reboot.
+
+- `sat bootsys boot` will more reliably determine if the `hms-discovery` cronjob
+  has been scheduled during the `cabinet-power` stage. This same functionality
+  is also used by `sat swap blade`.
+
+- SAT will use the BatchV1 Kubernetes API to manipulate CronJobs, instead of the
+  BatchV1Beta1 API as previously.
```

Comment:
The "as previously" is kind of awkward. Maybe just leave it off. @jld-dehn-hpe might have a suggestion.